lib/smooth/include/smooth/core/io/i2c/I2CMasterDevice.h

```diff
@@ -74,7 +74,8 @@ namespace smooth::core::io::i2c
             /// \return true on success, false on failure
             bool write(uint8_t address, std::vector<uint8_t>& data, bool expect_ack = true);
 
-            /// Reads data from the register of the slave with the provided address.
+            /// Reads data from the 8 bit register of the slave with the provided address.  This function DOES NOT
+            // support slave clock-stretching
```

Comment:
```suggestion
            /// support slave clock-stretching
```

Reply:
This is how uncrustify formats the document, but will change.  I think it looks better with three forward slashes then 2 forward slashes